src/components/Modal/index.js

```diff
@@ -228,7 +236,7 @@ class Modal extends React.Component {
             >
               <ModalContainer
                 small={getModalSize({ deviceSize, preferredSize: size })}
-                ref={this.containerRef}
+                // ref={this.containerRef}
```

Comment:
Do we no longer need a ref on `ModalContainer`?

Reply:
I'll delete it.